Userland/Services/WindowServer/ScreenLayout.h

```diff
@@ -28,6 +33,22 @@ class ScreenLayout {
             return { location, { resolution.width() / scale_factor, resolution.height() / scale_factor } };
         }
 
+        static String mode_to_string(Mode mode)
+        {
+#define ENUMERATE_MODE_ENUM(val) \
```

Comment:
Usually we prefix these with `__`, e.g. here: https://github.com/SerenityOS/serenity/blob/086969277e74d8ba065bf8145d3aeb0dec0bfee5/Userland/Libraries/LibJS/Token.cpp#L18-L27

Can probably return a StringView as well.